### PR TITLE
README: Add warning about terminal app accessing the same port.

### PR DESCRIPTION
**Pull request by pfalcon**

### CODE REVIEW

**Comment by pfalcon** [comment]
I'm hitting the issue regularly, the forgetting about it, then wondering
what's wrong. Wonder, how many people hit the same.

I submitted this earlier online via github's edit feature, wonder if it didn't work.

File: README.md [hunk 1]
```diff
@@ -269,7 +269,7 @@ ESP8266 problems can be fiddly to troubleshoot. Try the suggestions here if you'
 If you see errors like "Failed to connect to ESP8266" then your ESP8266 is probably not entering the bootloader properly:
 
 * Check you are passing the correct serial port on the command line.
-* Check you have permissions to access the serial port, and other software (such as modem-manager on Linux) is not trying to interact with it.
+* Check you have permissions to access the serial port, and other software (such as modem-manager on Linux) is not trying to interact with it. A common pitfall is leaving a serial terminal accessing this port open in another window and forgetting about it.
```
Comment (jrobeson) [inline README.md]: i think this deserves its own bullet point.

Comment (jrobeson) [inline README.md]: to clarify, I run into this so often, that I think it deserves its own bullet point :)

Comment (pfalcon) [inline README.md]: @jrobeson : You're welcome to rewrite it like you think will be better, while keeping it coherent, and submit new patch, I'll close this one then.

**Comment by projectgus** [comment]
@jrobeson , are you planning to rewrite or should I merge this as-is?

**Comment by jrobeson** [comment]
merge it. I now think that just the bit about permissions needs to go on its own line, thus keeping the gist of @pfalcon's patch intact.  I'll do that after this is merged.

**Pull request merged**